src/test/java/com/bakdata/common_kafka_streams/integration/CleanUpTest.java

```diff
@@ -164,6 +172,46 @@ void shouldReprocessAlreadySeenRecords() throws InterruptedException {
         this.runAndAssertSize(6);
     }
 
+    @Test
+    void shouldDeleteValueSchema() throws InterruptedException, IOException, RestClientException {
+        final SchemaRegistryClient client = this.schemaRegistryMockExtension.getSchemaRegistryClient();
+        this.app.setStreamsConfig(Map.of(
+                StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, SpecificAvroSerde.class.getName()
+        ));
+        final TestRecord testRecord = TestRecord.newBuilder().setContent("val 1").build();
+        final SendValuesTransactional<TestRecord> sendRequest = SendValuesTransactional
+                .inTransaction(this.app.getInputTopic(), Collections.singletonList(testRecord))
+                .with("schema.registry.url", this.schemaRegistryMockExtension.getUrl())
+                .with(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, KafkaAvroSerializer.class.getName())
+                .build();
+
+        this.kafkaCluster.send(sendRequest);
+        this.app.run();
+        this.softly.assertThat(client.getAllSubjects()).hasSize(1);
```

Comment:
please be more specific and make assertions such as assertThat(client.getAllSubjects()).contains("my-topic-value"). Also add a test were another schema is still in the schema registry after clean up, i.e., the schema registry is not wiped completely